src/components/Capture/withCameraDetection.js

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact'
+import { PureComponent } from 'preact-compat'
+import { checkIfHasWebcam } from '../utils'
+
+export default WrappedComponent =>
+  class WithCameraDetection extends PureComponent {
+    state = {
+      hasCamera: undefined,
+    }
+
+    componentDidMount(){
+      this.cameraChecker = setInterval(this.checkCameraSupport, 2000)
```

Comment:
Is this a new feature?

Reply:
No, just renamed `webcam` to `camera`

https://github.com/onfido/onfido-sdk-ui/blob/development/src/components/Capture/capture.js#L28